webapp/src/features/intersections/assessments/lane-direction-of-travel-assessment.tsx

```diff
@@ -112,31 +123,21 @@ export const LaneDirectionOfTravelAssessmentCard = (props: {
   }
 
   return (
-    <Grid2 height="500px">
-      <Card sx={{ height: '100%', overflow: 'visible' }}>
+    <Grid2 sx={{ height: '100%', minHeight: assessment === undefined ? 200 : 500 }}>
+      <Card sx={{ height: '100%', overflowX: 'auto' }}>
```

Comment:
Question: Setting overflowX here will cause both the title and graph to scroll - is that the intended behavior? Or should the title stay fixed?

Reply:
That is the intended behavior